qiita_pet/templates/study_ajax/add_artifact.html

```diff
@@ -0,0 +1,112 @@
+<script type="text/javascript">
+  // Global counter used to check if the form can be submitted or not
+  var error_count = 0;
```

Comment:
Is there any way we can try an avoid adding global variables? I know it sometimes hard, but this is a slippery slope and it has the potential of getting out of control.